tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -1563,4 +1570,80 @@ public function userGetsAllUsersOfTwoGroupsUsingTheGraphApi(string $user, string
 		);
 		$this->featureContext->setResponse($response);
 	}
+
+	/**
+	 * @When /^the administrator has given "([^"]*)" the role "([^"]*)" using the Graph API$/
+	 *
+	 * @param string $user
+	 * @param string $role
+	 *
+	 * @return void
+	 *
+	 * @throws GuzzleException
+	 * @throws Exception
+	 */
+	public function theAdministratorHasGivenTheRoleUsingTheGraphApi(string $user, string $role): void {
+		$admin = $this->featureContext->getAdminUserName();
+		$userId = $this->featureContext->getAttributeOfCreatedUser($user, 'id');
+		$userId = $userId ?? $user;
```

Comment:
```suggestion
		$userId = $this->featureContext->getAttributeOfCreatedUser($user, 'id') ?? user;
```
Can be done like this as well?